middleware/cors.go

```diff
@@ -0,0 +1,28 @@
+package middleware
+
+import (
+	"net/http"
+
+	"github.com/gorilla/handlers"
+	"github.com/raystack/raccoon/config"
+)
+
+var cors func(http.Handler) http.Handler
+
+func loadCors() {
```

Comment:
The way the handler.CORS(opt...) function is structured, I don't see having a mechanism to test whether right values have been passed to the function, since the value passed are of type functions, which do not have state to be asserted.

I read up a bit as well and it suggested trying to test whether the behaviour is according to expectation(ie integration testing)

Other than that, I can't think of ways to test it since, the cors middle doesn't take a config struct but an options array which cannot be asserted against. 

Ex: https://gist.github.com/wtask/fe18ceecc2d27295138f59ccdf075b78. The suggestion is test it in the package itself, ie in this case gorilla.

Reply:
ok